windows/internal/cuda_install.bat

```diff
@@ -185,22 +185,15 @@ echo Installing CUDA toolkit...
 7z x %CUDA_SETUP_FILE% -o"%SRC_DIR%\temp_build\cuda"
 pushd "%SRC_DIR%\temp_build\cuda"
 
-rem Why find ":", if no process, the output is INFO: No tasks are running which match the specified criteria.
-tasklist /fi "imagename eq msiexec.exe" | find ":"
-if %ERRORLEVEL% NEQ 0 (
-    echo There's another installer running.
-)
-
-start /wait setup.exe -s %ARGS%
+sc query wususerv
```

Comment:
typo? also check line 190

Reply:
I think both `sc query wususerv` could be removed

Reply:
Yeah, it's only used for debugging purposes